### PR TITLE
Finish CI integration

### LINKED ISSUE #21: Add CI
* [x] yarn lint
* [x] yarn test
* [x] manage.py runtests
* [x] flake8 or whatever the new thing is

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

**Comment by djmitche** [comment]
Note that this is based on #40 which is not yet reviewed.

File: requirements.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ asgiref==3.2.10
 Django==3.1.2
 djangorestframework==3.12.1
 django-csp==3.7
-pkg-resources==0.0.0
```
Comment (jwhitlock) [inline requirements.txt:5]: 👍 I had to do this locally as well

**Review by jwhitlock — APPROVED** [review]
Thanks @djmitche, I love getting linting integrated early!

File: requirements.txt (hunk 1 above)
Comment (djmitche) [reply requirements.txt:5]: I vaguely recall there is/was a pip bug where `pip freeze` produced this.  And I did create requirements.txt by running `pip freeze`.

**Pull request merged**